dev/com.ibm.ws.security.jaspic/bnd.bnd

```diff
@@ -35,13 +44,10 @@ Import-Package: \
   javax.xml.bind;version=!,\
   javax.xml.bind.annotation;version=!,\
   *
- 
-Private-Package: \
```

Comment:
Why did you remove this private package declaration?

Reply:
This was by accident. I noticed it in testing. Planned on fixing it when I merged. I will undo this change.